### DISCUSSION TITLE
is it possible to improve search on the docs site?

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#16015] [Ideas]
I think for example searching for "Custom Reports" should put [this page](https://pimcore.com/docs/platform/Pimcore/Tools_and_Features/Custom_Reports) above the fold:
![Screenshot 2023-09-27 at 17 12 01](https://github.com/pimcore/pimcore/assets/279826/b04bb499-044b-416d-ba75-5f0c189a3f66)
![Screenshot 2023-09-27 at 17 12 17](https://github.com/pimcore/pimcore/assets/279826/8e771ffc-d226-44d9-99b2-d10bbef76e51)

**Comment by SamyMP** [comment]
I agree, search is not always showing relevant results. I like the way the old documentation worked

**Comment by robertSt7** [comment]
Hi @NiklasBr thanks for the suggestion, I have created an issue for that. We will work on it as fast as we can.

**Reply by NiklasBr** [reply]
404 error on that link

**Reply by jdreesen** [reply]
It's probably a private repo.

**Reply by robertSt7** [reply]
Hi @NiklasBr, yes @jdreesen is right :)

**Reply by fashxp** [reply]
jfyi .. we are trying to integrate Algolia search

**Comment by fashxp** [comment]
Algolia is live now. Any feedback is appreciated.

**Reply by NiklasBr** [reply]
Showing all results defaults to content from `next` version rather than YYYY.mm one defined in the platform version.

Previous search term is not always saved the way it used to be, it was really convenient when the content of the page wasn't what I was looking for.

Otherwise it is a great improvement! 🔝👍
